components/page/body/index.vue

```diff
@@ -67,34 +74,35 @@
                 </div>
             </div>
             
-        </div>
-    
-        <div v-if="pageStore?.media?.length"  class="row mt-3">
-            <div class="col-12 col-md-3">
-                <h2 :style="pageTypeStyle" class="side-heading text-nowrap">{{t('Attachments')}} <span class="text-muted fs-4">({{pageStore?.media.length}})</span></h2>
+            <div v-if="pageStore?.media?.length"  class="row mt-3">
+                <div class="col-12 col-md-3">
+                    <h2 :style="pageTypeStyle" class="side-heading text-nowrap">{{t('Attachments')}} <span class="text-muted fs-4">({{pageStore?.media.length}})</span></h2>
```

Comment:
you have to put `?` on all sub element other risk of undefined error
```suggestion
                    <h2 :style="pageTypeStyle" class="side-heading text-nowrap">{{t('Attachments')}} <span class="text-muted fs-4">({{pageStore?.media?.length}})</span></h2>
```